docs/content/sidebars/operator.js

```diff
@@ -3,43 +3,70 @@
-            'operator/iota-full-node/source',
-            'operator/iota-full-node/monitoring-and-pruning',
+            'operator/full-node/overview',            
+            'operator/full-node/docker',
+            'operator/full-node/source',
+            {
+                type: 'category',
+                label: 'Full Node Configuration',
+                link: {
+                    type: 'doc',
+                    id: 'operator/full-node/configuration',
+                },
+                items: [
+                    'operator/full-node/configs/genesis',
+                    'operator/full-node/configs/network',
+                    'operator/full-node/configs/pruning',
+                    'operator/full-node/configs/snapshots',
+                    'operator/full-node/configs/archives',
+                ],
+            },
+            'operator/full-node/monitoring',
         ],
     },
-    'operator/validator-config',
-    'operator/data-management',
-    'operator/snapshots',
-    'operator/archives',
-    'operator/genesis',
-    'operator/indexer-functions',
     {
         type: 'category',
-        label: 'Validator Operation',
+        label: 'Validator Node',
         items: [
-            'operator/validator-operation/validator-tasks',
-            'operator/validator-operation/ansible/README',
-            'operator/validator-operation/docker/README',
-            'operator/validator-operation/systemd/README',
-        ]
+            'operator/validator-node/overview',            
+            'operator/validator-node/docker',
+            'operator/validator-node/systemd',            
+            {
+                type: 'category',
+                label: 'Validator Node Configuration',
+                link: {
+                    type: 'doc',
+                    id: 'operator/validator-node/configuration',
+                },
+                items: [
+                    'operator/validator-node/configs/genesis',
+                    'operator/validator-node/configs/network',
+                    'operator/validator-node/configs/pruning',
+                    'operator/validator-node/configs/snapshots',
+                    'operator/validator-node/configs/archives',
+                ],
+            },
+            'operator/validator-node/validator-tokenomics',
+            'operator/validator-node/cli-validator-command',
+            'operator/validator-node/monitoring',
+        ],
     },
-    'operator/validator-tool',
-    'operator/iota-tool',
     {
         type: 'category',
-        label: 'Node Monitoring and Metrics',
+        label: 'Extensions',
         items: [
-            'operator/telemetry/telemetry-subscribers',
-            'operator/telemetry/iota-metrics',
+            'operator/extensions/indexer-functions',
         ],
     },
+    'operator/data-management',
+    // 'operator/security-releases',
+    'operator/ssfn_guide',
+    'operator/activate_DoS_protection_for_validators',
```

Comment:
Shouldn't both elements be under the validator item?

Reply:
Moved. 07c041da6169d754e3dc8b5b574e6be522d262d4